app/models/user.rb

```diff
@@ -1,15 +1,29 @@
 class User < ActiveRecord::Base
-  attr_accessible :name, :oauth_expires_at, :oauth_token, :provider, :uid
-
-	def self.from_omniauth(auth)
-	  where(auth.slice(:provider, :uid)).first_or_initialize.tap do |user|
-	    user.provider = auth.provider
-	    user.uid = auth.uid
-	    user.name = auth.info.name
-	    user.oauth_token = auth.credentials.token
-	    user.oauth_expires_at = Time.at(auth.credentials.expires_at)
-	    user.save!
-	  end
-	end
-  
+  attr_accessible :provider, :uid, :name, :first_name, :last_name, :oauth_token,
+                  :oauth_expires_at, :city, :country, :description, :photo
+
+  has_attached_file :photo,
+  styles: {
```

Comment:
@Mywe4ka please change indentation to represent that ``` styles ``` and ``` default_url ``` is part of has_attached_file